app/assets/stylesheets/application.scss

```diff
@@ -73,4 +73,9 @@ footer {
   width: 100%;
   height: 2.5rem;    
   font-family: 'Special Elite'; 
```

Comment:
just an FYI... you should aways define a fallback font if for whatever reason your custom one doesn't load. https://www.w3schools.com/Css/css_font_fallbacks.asp